tests/unit/oidc/test_views.py

```diff
@@ -374,6 +371,73 @@ def test_mint_token_from_pending_trusted_publisher_invalidates_others(
     ]
 
 
+def test_mint_token_from_oidc_only_pending_publisher_fail(monkeypatch, db_request):
```

Comment:
This test tests an impossible state to reach as laid out in [this comment](https://github.com/pypi/warehouse/pull/14063/files#r1406874749) and I'm not sure what to do about it.